tests/examples/test_downloads.py

```diff
@@ -77,8 +77,8 @@ def test_file_copier(tmpdir):
 
 def test_local_file_cache(tmpdir):
     """Ensure that pyvista.examples.downloads can work with a local cache."""
-    basename = os.path.basename(examples.mapfile)
-    dirname = os.path.dirname(examples.mapfile)
+    basename = os.path.basename(examples.planefile)
+    dirname = os.path.dirname(examples.planefile)
```

Comment:
Why the switch from one to the other?

Reply:
Because this was loading the image file as a texture and `pv.Texture` is now in the `plotting` subpackage rather than `core` due to its libGL requirement. This allows us to test this module in the first stage of CI without graphics libs. Texture are well tested otherwise.